CONTRIBUTING.md

```diff
@@ -11,4 +13,48 @@ Although contributing to the project is allowed and even encouraged it has to fo
 
 
 
-This is a *guideline* which means that some exceptions may be made. However it is recommend that you respect it for your contribution to be validated. If you have any other question or think a point is unclear or too limiting feel free to contact an @admin
\ No newline at end of file
+This is a *guideline* which means that some exceptions may be made. However it is recommend that you respect it for your contribution to be validated. If you have any other question or think a point is unclear or too limiting feel free to contact an @admin
+
+## Installing an editing
+
+### Requirements
+
+You will need python 3 and the library `discord.py`.
+
+*(We would also recommend to work on Linux :penguin:)*
```

Comment:
why?

Reply:
Because we're make it on Linux x) It seems easier to me
And because I don't know how to run the .py on Windows x)

Reply:
I'd remove since I don't understand the goal of the restriction. Plus the CMD instructions are should be the same.